tests/generation/Bears.py

```diff
@@ -357,3 +357,17 @@ def test_filter_bears_ci_mode(self):
                                     if bear not in res_1[lang]]
                 for bear in additional_bears_by_lang[lang]:
                     self.assertIn(bear, additional_bears)
+
+    def test_print_coala_quickstart_version_short(self):
+        sys.argv.append('-v')
+        with self.assertRaises(SystemExit) as cm:
+            with retrieve_stdout() as custom_stdout:
+                main()
+                self.assertEquals('0.4.0', custom_stdout.getvalue)
```

Comment:
`main` raises an exception, causing to exit the `with self.assertRaises(...)` and never executes the `self.assertEqual`. The `assertEqual(...)` has to be moved out of the first `with`-statement:
```
with self.assertRaises(SystemExit) as cm:
    with retrieve_stdout() as custom_stdout:
        main()
self.assertEquals('0.4.0', custom_stdout.getvalue)
```

Reply:
Same below

Reply:
Thanks for pointing out that, I will amend it later